atomate/vasp/workflows/base/lobster.py

```diff
@@ -0,0 +1,219 @@
+            should be given with the full name and the correct capitalization.
+    Returns:
+    """
+
+    c = c or {}
+
+    vasp_cmd = c.get("VASP_CMD", VASP_CMD)
+    lobster_cmd = c.get("LOBSTER_CMD", LOBSTER_CMD)
+    db_file = c.get("DB_FILE", DB_FILE)
+
+    fws = []
+    # get the relevant potcar files!
+    inputset = LobsterSet(structure, address_basis_file=address_max_basis,
+                          user_incar_settings=user_incar_settings,
+                          user_kpoints_settings=user_kpoints_settings, isym=isym)
+    # get the basis from dict_max_basis
+    potcar_symbols = inputset.potcar_symbols
+    max_basis = Lobsterin._get_basis(structure=structure,
+                                     potcar_symbols=potcar_symbols, address_basis_file=address_max_basis)
+    min_basis = Lobsterin._get_basis(structure=structure,
+                                     potcar_symbols=potcar_symbols, address_basis_file=address_min_basis)
+
+    # get information about how many lobster calculations have to be performed to test all possible basis functions
+    all_basis = get_all_possible_basis_combinations(min_basis=min_basis, max_basis=max_basis)
+
+    staticfw = StaticFW(structure=structure,
+                        vasp_input_set=inputset,
+                        vasp_cmd=vasp_cmd, db_file=db_file, name="static")
+    fws.append(staticfw)
+
+    # append all lobster calculations that need to be done
+    fws_lobster = []
+    for ibasis, basis in enumerate(all_basis):
+        basis_dict = {}
+
+        for iel, elba in enumerate(basis):
+            basplit = elba.split()
+            basis_dict[basplit[0]] = " ".join(basplit[1:])
+
+        fws_lobster.append(
+            LobsterFW(structure=structure, parents=staticfw, calculationtype=calculationtype,
+                      delete_wavecar=delete_all_wavecars,
+                      delete_wavecar_previous_fw=False, lobster_cmd=lobster_cmd,
+                      db_file=db_file, user_supplied_basis=basis_dict,
+                      lobsterin_key_dict=user_lobsterin_settings,
+                      handler_group="default", validator_group="strict",
+                      name="lobster_calculation_{}".format(ibasis),
+                      lobstertodb_kwargs={"additional_fields": {"basis_id": ibasis,
+                                                                "number_lobster_runs": len(all_basis)}},
+                      additional_outputs=additional_outputs))
+
+    fws.extend(fws_lobster)
+    # wavecar from static run is deleted, WAVECARs without symmetry can be huge!
+    if delete_all_wavecars:
+        t = DeleteFilesPrevFolder(files=["WAVECAR", "WAVECAR.gz"], calc_loc="static")
+        final_fw = Firework([t], parents=fws_lobster, name="DelteWavecar")
+        fws.append(final_fw)
+
+    workflow = Workflow(fws, name="LobsterWorkflow")
+    return workflow
+
+
+def get_all_possible_basis_combinations(min_basis: list, max_basis: list) -> list:

```

Comment:
What is the rationale for including the basis information in atomate? I think this would be better suited for pymatgen  in the same location as the Lobsterin class as people might want to use this functionality without using atomate.

What do you think about moving it there?

Reply:
moving this to pymatgen.